hack/update-munge-docs.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+git_upstream=$(kube::util::git_upstream_remote_name)
```

Comment:
ah yeah, the part I was objecting to wasn't clobbering the env var so much as it's no longer possible to set the option to a custom value at all. If you don't think that's ever a reasonable use case then I'm fine with this.